addons/l10n_cn/data/account.account.template.csv

```diff
@@ -1,77 +1,184 @@
-"l10n_cn_4001","Paid in capital","4001","equity","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_4002","Capital Surplus","4002","equity","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_4003","Other Comprehensive Income","4003","equity","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_4101","Surplus Reserve","4101","equity","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_4103","Profit for the year","4103","equity","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_4104","Profit distribution","4104","equity","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_5001","Production Costs","5001","expense_direct_cost","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_5101","Manufacturing Expenses","5101","expense_direct_cost","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_5201","Service Cost","5201","expense_direct_cost","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_5301","R & D expenditure","5301","expense_direct_cost","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6001","Main Business Income","6001","income","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6051","Other Business Income","6051","income_other","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6101","Gains and Losses of fair value change","6101","income_other","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6111","Income from investment","6111","income_other","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6301","Non-operating Income","6301","income_other","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6401","Main Business Cost","6401","expense","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6402","Other Operating Costs","6402","expense","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6403","Operating Taxes and Surcharges","6403","expense","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6601","Selling Expenses","6601","expense","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6602","Management Expenses","6602","expense","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6603","Financial Expenses","6603","expense","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6701","Assets impairment Loss","6701","expense","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6711","Non-operating expenses","6711","expense","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6801","Income Tax Expense","6801","expense","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6901","Prior year income adjustment","6901","expense","l10n_cn.l10n_chart_china_small_business","False"
+"id","name","code","l10n_cn_type","chart_template_id/id","reconcile",parent_id/id
+"l10n_cn_1001","库存现金","1001","asset_cash","l10n_cn.l10n_chart_china_small_business","False",
```

Comment:
We avoid using localized terms in our data files like it was done before.
The standard way of handling it is to keep the names in English in the files and provide the translations in the translation files.

Reply:
localized  should be better understood for developer

Reply:
If the developer can read it :)
Odoo is maintained by people from around the world, and most people can read and write in English which is why it is the language that should be used for code/comments/data by default.